#! /bin/sh /usr/share/dpatch/dpatch-run
## rcon_tool_debianization.dpatch by Andres Mejia <mcitadel@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to modify /Tools/LinuxScripts/rcon script for use in a Debian based distro.
## This patch will work provided that the source tree was checked out to a directory named alienarena2007.

@DPATCH@
diff -urNad alienarena2007~/Tools/LinuxScripts/rcon alienarena2007/Tools/LinuxScripts/rcon
+++ Tools/LinuxScripts/rcon	2007-05-14 23:37:17.000000000 -0400
@@ -22,6 +22,10 @@
     Tony Jackson can be contacted at tonyj@cooldark.com
 	
 =end
+=begin
+	This script has been patched from its original source for distribution
+	with Debian systems.
+=end
 
 require 'socket'
 
@@ -31,7 +35,7 @@
 		return []
 	end
 
-	query = '����rcon ' + password + ' ' + message + ' ' + "\x00"
+	query = '����rcon ' + password + ' ' + message + ' ' + "\x00"
 	connections = Hash.new
 	failed = Array.new
 	
@@ -103,10 +107,10 @@
 	puts 'RCon utility for Alien Arena servers'
 	puts ''
 	puts '  Usage:'
-	puts '  rcon <server:port> password <commandstring>'
+	puts '  alienarena2007-server -r <server:port> password <commandstring>'
 	puts '  Optional arguments:'
 	puts ''
-	puts '  --help  This message'
+	puts '  alienarena2007-server -r --help  This message'
 	puts ''
 	exit
 end
